project/Build.scala

```diff
@@ -1452,6 +1438,7 @@ object Build {
           name := "JavaLib Ex Test Suite",
           publishArtifact in Compile := false,
           testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s"),
+          scalacOptions += "-Ynowarn-default-junit-methods",
```

Comment:
Yes, I checked that all the methods that get the warning are executing. Additionally if the plugin does not run the test would just fail with linking errors.